dashboard/src/components/Sidebar.vue

```diff
@@ -51,7 +54,7 @@ onMounted(() => {
               class="d-block py-3 text-decoration-none sidebar-link"
               active-class="active"
               exact-active-class="exact-active"
-              :to="{ name: item.routeName }"
+              :to="item.route"
```

Comment:
same here

Reply:
Similarly, `menuItems` now contains the actual route object so we don't need to build a new one after the string.